### DISCUSSION TITLE
Converter

### DISCUSSION THREAD

**Discussion by Npa972** [DeaDBeeF-Player/deadbeef#2831] [Q&A]
Hi,
Just a question.
I wanted to convert flac files to ogg using converter (ogg vbr 320).
It doesn't work (oggenc missing).
I have to install vorbis-tools to convert.

My question : why vorbis-tools is not in the dependencies ?

System : archlinux
And thanks for this great player !

**Comment by Oleksiy-Yakovenko** [comment]
Mainly, because deadbeef can work without any encoders, and without converter, so making it a dependency will force people who don't need it to install many unnecessary things. Think of any other possible decoders etc.
